The Flask and FastAPI framework (seminars)/Урок 2. Погружение во Flask/templates/welcome.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Добро пожаловать</title>
+</head>
+<body>
+    <h1>Добро пожаловать, {{ name }}!</h1>
+    <form action="/logout" method="get">
```

Comment:
И тут, но метод должен быть `post`
```suggestion
    <form method="post">
```